manifest_staging/charts/secrets-store-csi-driver/values.yaml

```diff
@@ -120,7 +120,7 @@ syncSecret:
 minimumProviderVersions:
 
 ## ; delimited list of providers that support grpc for driver-provider [alpha]
-grpcSupportedProviders:
+grpcSupportedProviders: gcp;
```

Comment:
Should we include all the grpc supported providers here?

Reply:
The reason we haven't included is because azure provider versions prior to `0.0.8` still work with the current driver. If the user wishes to use the latest provider they set the args.

Reply:
That makes sense!